src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/TestAzureRmDataLakeStoreAccount.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.DataLakeStore.Models;
+
+namespace Microsoft.Azure.Commands.DataLakeStore
+{
+    [Cmdlet(VerbsDiagnostic.Test, "AzureRmDataLakeStoreAccount"), OutputType(typeof (bool))]
+    public class TestAzureDataLakeStoreAccount : DataLakeStoreCmdletBase
+    {
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            HelpMessage = "Name of a specific account.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = false,
+            HelpMessage = "Name of resource group under which want to test the account.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        protected override void ProcessRecord()
+        {
+            try
+            {
+                DataLakeStoreClient.GetAccount(ResourceGroupName, Name);
+                WriteObject(true);
+            }
+            catch
```

Comment:
again, catching everything is not a good practice, would be better to catch specific exception. For example, if there is a null ref exception happened it will be hard to find it out as this block will swallow it.